### PR TITLE
GH866: Update.NET Core SDK 2.1 to 2.1.502

### LINKED ISSUE #866: Update.NET Core SDK 2.1 to 2.1.502
If you install `dotnet-sdk-2.1` on Mac, Linux and Windows you get  `2.1.502`

If you pull `microsoft/dotnet:2.1-sdk` you'll get `2.1.500`

### PR DESCRIPTION
**Pull request by devlead**
* fixes #866

### CODE REVIEW

**Comment by MichaelSimons** [comment]
@dotnet-bot test NanoServer-1803 - 3.0 Dockerfiles please

File: 2.1/sdk/stretch/arm32v7/Dockerfile [hunk 1]
```diff
@@ -14,18 +14,18 @@ RUN apt-get update \
...
-    && dotnet_sha512='106d3e5b7b96ca3522e78b77f094cecaac38c348d7138c2fcae0464ff9d57ec137802603c4a5c6f81b9eae9de39b86bff8d9fc573817e033982eb576532e334c' \
+    && dotnet_sha512='f5a9bf61c81e0bc5a33367f63320c792680e80dede884c604531715bfb741f4b0f53779ba57ed8e0a2ad3546358a42f531fb28952725e17f8cb5ca3a4b66ac60' \
     && echo "$dotnet_sha512 dotnet.tar.gz" | sha512sum -c - \
     && mkdir -p /usr/share/dotnet \
     && tar -zxf dotnet.tar.gz -C /usr/share/dotnet \
     && rm dotnet.tar.gz \
     && ln -s /usr/share/dotnet/dotnet /usr/bin/dotnet \
     # Add NuGet cache (ARM SDK doesn't include it)
     && curl -SL --output /usr/share/dotnet/sdk/$DOTNET_SDK_VERSION/nuGetPackagesArchive.lzma https://dotnetcli.blob.core.windows.net/dotnet/Sdk/$DOTNET_SDK_VERSION/nuGetPackagesArchive.lzma \
-    && lzma_sha512='329b72d6576a200e2bf75297102806d33c691f38489c3b2b83fe354509cbc19bf9509a219b2b0a5e3a3a193326ef8a8f836ee11cd1df196e75b4b3738e783b15' \
```
Comment (MichaelSimons) [inline 2.1/sdk/stretch/arm32v7/Dockerfile]: I'm not seeing that the lzma artifact changed, it appears identical to 2.1.500 looks to have the same sha.  Where did you grab this new sha from?

Comment (devlead) [reply 2.1/sdk/stretch/arm32v7/Dockerfile]: I did a 
```bash
curl -SL --output/nuGetPackagesArchive.lzma https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.1.502/nuGetPackagesArchive.lzma
sha512sum nuGetPackagesArchive.lzma
329b72d6576a200e2bf75297102806d33c691f38489c3b2b83fe354509cbc19bf9509a219b2b0a5e3a3a193326ef8a8f836ee11cd1df196e75b4b3738e783b15
echo "329b72d6576a200e2bf75297102806d33c691f38489c3b2b83fe354509cbc19bf9509a219b2b0a5e3a3a193326ef8a8f836ee11cd1df196e75b4b3738e783b15 nuGetPackagesArchive.lzma" |sha512sum -c
nuGetPackagesArchive.lzma: OK
```

Comment (MichaelSimons) [reply 2.1/sdk/stretch/arm32v7/Dockerfile]: I'm confused.  Why are you proposing to change it to `b2bb96a26d6951bad877faace16faa4f8fe1a52ff0858fdf869294f7b230df35db08715eaf8a0ec8434c07b2036d5bb1640c5fc443d2654af322c14ecf95377d`?

Comment (devlead) [reply 2.1/sdk/stretch/arm32v7/Dockerfile]: Hmm I'm unsure, I'll revert that one.

Comment (MichaelSimons) [reply 2.1/sdk/stretch/arm32v7/Dockerfile]: FYI - It exists in two places. Here is the [bionic-arm32v7 Dockerfile](https://github.com/dotnet/dotnet-docker/pull/867/files#diff-ea7107e2fcb24417dc07861e2dc4b377R28)

Comment (devlead) [reply 2.1/sdk/stretch/arm32v7/Dockerfile]: Yip, addressed both.

**Pull request merged**